client/src/components/ProductCard.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from "prop-types";
 import HairbrushImg from "../assets/images/hairbrush.jpg";
 
 const Container = styled.div`
```

Comment:
your product container disappears behind the bottom nav ( just a little bit) . you might wanna add some padding there :)

Reply:
ah,  true! Didn't pay attention to that! Thanks :)